<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>356</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Parametrs</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>506</width>
             <height>202</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="horizontalSpacing">
               <number>5</number>
              </property>
              <property name="verticalSpacing">
               <number>3</number>
              </property>
              <item row="11" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Output folder</string>
                </property>
               </widget>
              </item>
              <item row="10" column="1">
               <widget class="QToolButton" name="toolButton_layer_refresh">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>refresh.png</normaloff>refresh.png</iconset>
                </property>
               </widget>
              </item>
              <item row="13" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLineEdit" name="txtPathToProtocols"/>
                </item>
               </layout>
              </item>
              <item row="10" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_19">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QComboBox" name="cmbLayers">
                  <property name="mouseTracking">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="8" column="0">
               <widget class="QComboBox" name="cmbLayersRoad"/>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Layer of buildings (points/polygons)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QToolButton" name="toolButton_GTFS">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QToolButton" name="toolButton_layer_road_refresh">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normalon>refresh.png</normalon>
                 </iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLineEdit" name="txtPathToGTFS"/>
                </item>
               </layout>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Layer of roads</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>GTFS folder</string>
                </property>
               </widget>
              </item>
              <item row="13" column="1">
               <widget class="QToolButton" name="toolButton_protocol">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="14" column="0">
               <widget class="QCheckBox" name="cbRunCalcFootPathRoad">
                <property name="text">
                 <string>Run calculation foot path on road</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextBrowser" name="textLog"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPlainTextEdit" name="textInfo">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBreakOn">
       <property name="text">
        <string>Break</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="gridFrame">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblMessages">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textInfo</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnBreakOn</tabstop>
  <tabstop>textLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
